frontend/src/app/shared/format/date-format.ts

```diff
@@ -0,0 +1,17 @@
+import { MatDateFormats } from '@angular/material/core';
+
+export const GLOBAL_DATE_FORMAT = 'dd.MM.yyyy';
+
+export const CUSTOM_LUXON_DATE_FORMATS: MatDateFormats = {
+  parse: {
+    dateInput: GLOBAL_DATE_FORMAT
+  },
+  display: {
+    dateInput: GLOBAL_DATE_FORMAT,
+
+    monthYearLabel: 'MM yyyy',
+
+    dateA11yLabel: 'LL',
+    monthYearA11yLabel: 'MM yyyy'
```

Comment:
```suggestion
  display: {
    dateInput: GLOBAL_DATE_FORMAT,

    monthYearLabel: 'MMMM yyyy',

    dateA11yLabel: 'LL',
    monthYearA11yLabel: 'MMMM yyyy'
```
I'd prefer this being `MMMM` as `10 2025` looks weird too me. Also for the `monthYearA11yLabe` writing the month out might be better anyway. `MMMM` will display as `October` or `Oktober` respectively.

If you add these two things into `app.config.ts`, the month should be auto translated too:
- `{provide: LOCALE_ID, useValue: 'de-CH' }`
- `registerLocaleData(localeDeCH);`

Both might need some additional imports:
```
import { registerLocaleData } from '@angular/common';
import localeDeCH from '@angular/common/locales/de-CH';
import { LOCALE_ID } from '@angular/core';
```

Nice job on the `LL` though! I did not know that and it is nice to learn something new!